_spinnaker_install_admin_guides/vault-configuration.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: post
+title: Configuring Vault for the Kubernetes Auth
```

Comment:
either "Configuring Vault for the Kubernetes Auth Method" or "Configuring Vault for Kubernetes Auth"?